### DISCUSSION TITLE
accessing a process rate parameter

### DISCUSSION THREAD

**Discussion by wilbertdew** [Open-Systems-Pharmacology/Forum#986] [Questions & Problems] [answered]
Hi All,

I would like to use the process rate parameter for endosomal clearance in the reactions building block, is this possible?
I have checked the "create process parameter" checkbox, and I can see the process rate parameters in each organ in the simulation (under neighborhoods) and I can also plot them. However I cannot find it in the reactions building block. Does anyone know how I can do this? I can find the process rate parameter in the reactions building block for the paracellular absorption, which is a transport that has already the process rate parameter generated in the default large molecule mice model that I exported to Mobi. I believe the issue is that I need to update the reactions building block in such a way that it recognizes what I have done in the passive transport building block. I have already tried to do this by creating a new simulation but no luck. Thanks in advance for the help!
Wilbert

P.S. I am working with version 8

**Comment by Yuri05** [comment, marked as answer]
`ProcessRate `parameters are created in the **simulations** only, not in the building blocks.
However you still can use them in the formulas, but you need to create such a reference **manually**.

For this: right click on the References area of the formula and select `Create Reference...`
![grafik](https://user-images.githubusercontent.com/25061876/136423752-32e4d15a-25e0-4bfc-9de3-3a0d945517d3.png)

This will create an empty reference like below:
![grafik](https://user-images.githubusercontent.com/25061876/136423866-79d299e6-1926-40dc-a41c-0016664b4b4e.png)

Now enter the alias and the path of your ProcessRate-parameter manually and set the dimension.
* **Path** can be absolute or relative
* **Dimension** is either `Amount per time` (for amount based projects) or `Concentration per time` (for conc based projects)

Attached you can find an example with 3 reactions. Reactions **RE2** and **RE3** use the process rate parameter of the reaction **RE1** in their kinetics definition (in **RE2** it's referenced via **absolute** path, in **RE3** via **relative** path).

[Example_ProcessRate.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/7306685/Example_ProcessRate.zip)

<img width="450" src="https://user-images.githubusercontent.com/25061876/136455335-dfa307eb-6d54-4157-bacf-249c3fcdac71.png">

<img width="400" src="https://user-images.githubusercontent.com/25061876/136455401-6aed268c-77ad-4151-8eda-2a5d485cc5cb.png"><img width="400" src="https://user-images.githubusercontent.com/25061876/136455444-af4cb5e2-51ee-4daf-9b15-a7f7ade57973.png">

**Reply by wilbertdew** [reply]
Thanks @Yuri05, that was really helpful!
